### PR TITLE
Swap merge args, rename config.entries --> config.include

### PR DESCRIPTION
**Pull request by willemneal**
Also make transforms relative or use require

### CODE REVIEW

**Comment by jtenner** [comment]
I discussed this with dcode and he said we should use entries, and not include, even though include matches typescript ergonomics.

Feel free to talk this over with dcode and maxgraey, and I won't close this pr.

**Comment by willemneal** [comment]
I only change that because you used includes in your examples.  I'm fine changing to entries.

Also I want to add a `workspace` field, but again feel like perhaps it should be a separate tool.

**Comment by willemneal** [comment]
Also I tested this with NEAR projects and it works like a charm!  Which means I don't have to support a separate compile tool.

**Comment by jtenner** [comment]
Yeah this looks great! I forgot about transform paths :).

File: cli/asc.js [hunk 1]
```diff
@@ -283,10 +283,19 @@ exports.main = function main(argv, options, callback) {
   while (asconfig) {
     // merge target first, then merge options, then merge extended asconfigs
     if (asconfig.targets && asconfig.targets[target]) {
-      args = optionsUtil.merge(exports.options, args, asconfig.targets[target]);
+      args = optionsUtil.merge(exports.options, asconfig.targets[target], args);
```
Comment (jtenner) [inline cli/asc.js:286]: Why was this changed?

Comment (jtenner) [reply cli/asc.js:286]: I switched it back, because targets should be respected over default "options".
